src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.spec.ts

```diff
@@ -36,11 +36,13 @@ describe('RgwBucketListComponent', () => {
     rgwBucketServiceListSpy.and.returnValue(of(null));
```

Comment:
This change is required for the tests to pass:
```suggestion
    rgwBucketServiceListSpy.and.returnValue(of([]));
```